spec/features/admin_user/edit_item_spec.rb

```diff
@@ -0,0 +1,56 @@
+require "rails_helper"
+
+RSpec.feature "the admin can create new items", type: :feature do
+
+    let(:dessert) { Category.create(name: "dessert") }
+
+    let(:unicorn) { Item.create(title: "Unicorn on the Cob",
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.